sdk/src/wallet/core/operations/stronghold_backup/mod.rs

```diff
@@ -69,7 +69,7 @@ impl Wallet {
     /// coin type doesn't match
     /// if ignore_if_bech32_hrp_mismatch == Some("rms"), but addresses have something different like "smr", no accounts
     /// will be restored.
-    pub async fn restore_backup(
+    pub async fn restore_from_backup(
```

Comment:
Can't do this breaking change in develop

Reply:
ah, shoot, it's develop, right